perf/aiostress.py

```diff
@@ -20,12 +20,12 @@ def setUp(self):
         Source:
          https://oss.oracle.com/~mason/aio-stress/aio-stress.c
         """
-        aiostress_c = self.params.get('aiostress_c', default='aio-stress.c')
-        c_path = self.get_data_path(aiostress_c)
-        shutil.copy(c_path, self.srcdir)
+        aiostress = self.fetch_asset('https://oss.oracle.com/~mason/aio-stress/aio-stress.c')
+        shutil.copy(aiostress, self.srcdir)
         os.chdir(self.srcdir)
         # This requires libaio.h in order to build
-        process.run('gcc -Wall -laio -lpthread -o aio-stress %s' % aiostress_c)
+        process.run('gcc -Wall -laio -lpthread -o aio-stress %s' %
+                    os.path.basename(aiostress))
```

Comment:
I was suspicious of how/why `os.path.basename` was being used here. Turns out it made the test fail. If you think about it, the cached asset will not be in `self.srcdir`. The fix:

```diff
diff --git a/perf/aiostress.py b/perf/aiostress.py
index 9229171..6217200 100755
--- a/perf/aiostress.py
+++ b/perf/aiostress.py
@@ -24,8 +24,7 @@ class Aiostress(Test):
         shutil.copy(aiostress, self.srcdir)
         os.chdir(self.srcdir)
         # This requires libaio.h in order to build
-        process.run('gcc -Wall -laio -lpthread -o aio-stress %s' %
-                    os.path.basename(aiostress))
+        process.run('gcc -Wall -laio -lpthread -o aio-stress %s' % aiostress)
 
     def test(self):
         """
```